apps/antalmanac/src/stores/AppStore.ts

```diff
@@ -6,6 +6,14 @@ import { Schedules } from './Schedules';
 import { SnackbarPosition } from '$components/NotificationSnackbar';
 import { CalendarEvent, CourseEvent } from '$components/Calendar/CourseCalendarEvent';
 import { useTabStore } from '$stores/TabStore';
+import { saveSchedule } from '$actions/AppStoreActions';
+
+type ActionType = [
+    null | string,
+    null | number | string | RepeatingCustomEvent | ScheduleCourse,
+    null | number | string | number[],
+    null | number | string | RepeatingCustomEvent,
+];
```

Comment:
As we discussed in the sprint, it's best to handle this separately and be more explicit